### PR TITLE
Update getting started to 6.0.1

### PR DESCRIPTION
**Pull request by klizhentas**
Simplify getting started guide to use ACME and new
teleport configure command

### CODE REVIEW

**Comment by klizhentas** [comment]
@benarent can you please give this version a go and update the video guide?

File: docs/pages/getting-started.mdx [hunk 1]
```diff
@@ -56,109 +61,43 @@ cd teleport
...
-
-Otherwise you can start Teleport directly:
-
-`sudo teleport start`
+Start Teleport:
 
-You can access Teleport's web UI on port 3080.
-Replace `teleport.example.com` with your domain: `https://teleport.example.com:3080/`
+```bash
+sudo teleport start
```
Comment (benarent) [inline docs/pages/getting-started.mdx:96]: I would suggest we recommend starting with -d as it provides better debug info when getting started. 
```suggestion
sudo teleport start -d
```

**Review by benarent — APPROVED** [review]

File: docs/pages/getting-started.mdx [hunk 2]
```diff
@@ -56,109 +61,43 @@ cd teleport
...
-    --manual-public-ip-logging-ok \
-    --email foo@example.com \
-    -d "teleport.example.com, *.teleport.example.com"
-  ```
-</Admonition>
-
-**Update `teleport.yaml`**<br />
-Once you've obtained certificates from Let's Encrypt, the below commands will update Teleport's
-config file to use your newly configured domain and TLS certificates.
+To get started use an `A` or `CNAME` record pointing to the IP/FQDN of the machine with Teleport installed.
```
Comment (benarent) [inline docs/pages/getting-started.mdx:83]: Should we also add a note to recommend adding a wildcard subdomain `*.tele` for Teleport to work with Teleport Application Access?

Comment (klizhentas) [reply docs/pages/getting-started.mdx:83]: Good point, will add

File: docs/pages/getting-started.mdx (hunk 1 above)
Comment (webvictim) [reply docs/pages/getting-started.mdx:96]: Yeah, if we're not emitting enough information to be useful on first startup at `INFO` level then we should fix that in Teleport.

**Review by webvictim — APPROVED** [review]

**Review by r0mant — APPROVED** [review]

**Pull request merged**